prisma/schema.prisma

```diff
@@ -0,0 +1,46 @@
+// This is your Prisma schema file,
+// learn more about it in the docs: https://pris.ly/d/prisma-schema
+
+// TODO: use format schema to format this && break the build
+// if it's not formatted correctly
+// https://github.com/prisma/prisma/issues/3265
+
+datasource db {
+  provider = "postgresql"
+  url      = env("DATABASE_URL")
+}
+
+generator client {
+  provider        = "prisma-client-js"
+  previewFeatures = ["interactiveTransactions"]
+}
+
+model User {
+  id                   String                 @id @default(uuid())
+  createdAt            DateTime               @default(now())
```

Comment:
Prisma deals with this without us having to do anything. And this how the prisma docs outline the examples. 

Prisma quotes as needed while executing the queries and so on.

The one advantage I see is with switching to snake case would make it slightly easier to add custom sql migrations (because of the hassle of the quoting). What do you think?

Reply:
The other (in my mind, more pressing) advantage that comes to mind is if we want to run our own queries directly against the database when developing or debugging. With camel case, we'll need to remember to always quote the identifiers.

If it were just custom migrations, I'd say that's not something we plan on doing and we should just stick with camel case. But I can foresee us running custom queries fairly regularly.